pkg/watcher/resource-event-handler/pod.go

```diff
@@ -34,9 +34,9 @@ func (p *podEventHandler) GetResourceObject() runtime.Object {
 }
 
 func (p *podEventHandler) OnAdd(obj interface{}) {
-	glog.V(4).Infof("OnAdd(): pod %v", obj)
+	log.Debug().Msgf("OnAdd(): pod %v", obj)
```

Comment:
remove first log message ?

Reply:
the first log message, print the complete pod data, the second just print the name and the namespace